src/freebsd_sysctl.c

```diff
@@ -27,6 +27,59 @@
 
 int system_pagesize = PAGE_SIZE;
 int number_of_cpus = 1;
+#if __FreeBSD_version >= 1200029
```

Comment:
Is this a typo?
They removed the structure after this version?

Reply:
They changed and hid it. They also defined `__vmmeter` as a workaround in `vmstat` utility.